### PR TITLE
Sync Book Store instructions to JS track

### PR DESCRIPTION
**Pull request by safwansamsudeen**
@SleeplessByte, I've synced the updated Book Store instructions to this track, with no other changes. This PR is per your comment on [this forum topic](https://forum.exercism.org/t/improve-instructions-of-book-store-exercise/4767/23). Thanks!

### CODE REVIEW

**Comment by safwansamsudeen** [comment]
Changed the append file as mentioned in the forum too.

File: exercises/practice/book-store/.docs/instructions.md [hunk 1]
```diff
@@ -33,36 +26,36 @@ For example, how much does this basket of books cost?
...
 - 5 books at a 25% discount
-- +3 books at a 10% discount
+- 3 books at a 10% discount
 
 Resulting in:
 
-- 5 _ (8 - 2.00) == 5 _ 6.00 == $30.00
-- +3 _ (8 - 0.80) == 3 _ 7.20 == $21.60
+- 5 × (100% - 25%) * $8 = 5 × $6.00 = $30.00, plus
+- 3 × (100% - 10%) * $8 = 3 × $7.20 = $21.60
```
[the remark this answers is not in the document]
Comment (safwansamsudeen) [reply exercises/practice/book-store/.docs/instructions.md:40]: I've instead changed `*` to `x`, so that all three multiplication operators are the same.

**Comment by safwansamsudeen** [comment]
@SleeplessByte just a reminder about this.

**Review by SleeplessByte — APPROVED** [review]
Thank you!

**Pull request merged**